.github/workflows/detect-yml-change.yml

```diff
@@ -0,0 +1,28 @@
+name: Detect GH Actions Changes
+on:
+  pull_request:
+    types: [synchronize, reopened]
```

Comment:
Missing `opened` in the list for freshly opened PRs. Also, we could just leave this off since defaults is what I think we want.

```By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened```

Reply:
perfect thanks